modules/cli/src/main/scala/scala/cli/commands/fix/Fix.scala

```diff
@@ -217,11 +226,21 @@ object Fix extends ScalaCommand[FixOptions] {
   def removeDirectivesFrom(
     position: Option[Position.File],
     toKeep: Seq[StrictDirective] = Nil
+  )(
+    using loggingUtilities: LoggingUtilities
   ): Unit = {
     position match {
       case Some(Position.File(Right(path), _, _, offset)) =>
-        val keepLines   = toKeep.mkString("", newLine, newLine * 2)
-        val newContents = keepLines + os.read(path).drop(offset).stripLeading()
+        val keepLines    = toKeep.mkString("", newLine, newLine * 2)
+        val newContents  = keepLines + os.read(path).drop(offset).stripLeading()
+        val relativePath = loggingUtilities.relativePath(path)
```

Comment:
What path will be printed if users pass:
```
$ echo $pwd
/tmp/scala-demo
$ scala-cli fix Hello.scala /User/scala-cli/project/scala-cli/Hello.scala
```

Is it proper to show a relative path for the second path?

Reply:
Good point, the second path gets turned into an ugly `../../User/scala-cli/project/scala-cli/Hello.scala` so a fix is needed.